.changeset/tame-stingrays-sit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/eleventy-plugin-lit': patch
+---
+
+Fix using Lit SSR with Eleventy's Serverless Plugin #3074
```

Comment:
Let's make the changelog a bit more descriptive of what's beging fixed. The PR number will be automatically added by the changeset utility.
```suggestion
Fix transform breakage in situations where `outputPath` is false (e.g. setting `permalink: false` or using the serverless plugin).
```

Reply:
Perfect! Thanks for the feedback!